src/view/HenvendelseList.tsx

```diff
@@ -0,0 +1,16 @@
+import {HenvendelseData} from "../utils/typer";
+import React from "react";
+import {Henvendelse} from "./Henvendelse";
+
+
+interface Props {
+    henvendelseDataList: HenvendelseData[]
+
+}
+
+export function HenvendelseList(props: Props) {
+    return <> {props.henvendelseDataList.reverse().map( (henvendelseData, i) => <div key={i}>
```

Comment:
Kan være github som tuller for formatteringen, men tror det er litt å hente på å legge inn noen linjeskift her. 

F.eks
```
return (
        <>
            {props.henvendelseDataList
                .reverse()
                .map((henvendelse) => <Henvendelse henvendelseData={henvendelse}/>)
            }
        </>
    );
```

Som en side-note, så bør man unngå å bruke `i`(indeks) som `key` om det er mulig pga optimaliseringer som gjøres av react. Ikke sikkert det har så mye å si i dette tilfellet. Men om mulig så ville jeg byttet til `henvendelsesData.id` (eller tilsvarende). `div`-elementet er heller ikke nødvendig her tenker jeg, om det er behov for en `div` så burde den evt komme fra `Henvendelse`-komponenten

Reply:
Ja, stemmer det! Det snakket jeg og Alexander om også! Fikser det 😊